.github/workflows/check.yml

```diff
@@ -100,6 +100,26 @@ jobs:
       - x64
       - sre
     steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+
+      - name: 🔨 Build
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release -p simnode
+
+      - name: Simnode Test
+        run: |
+          YDATE=$(date -d yesterday +'%m-%d-%Y')
+          FILENAME=cl-1-$YDATE.zip
+          GS_BUCKET="composable-picasso-data-sync"
+          sudo gsutil cp gs://$GS_BUCKET/$FILENAME .
+          sudo unzip $FILENAME -d  /tmp/db
+          ./target/release/simnode --chain=picasso --base-path=/tmp/db --wasm-runtime-overrides=./target/release/picasso_runtime.compact.wasm --pruning=archive --execution=wasm
```

Comment:
I think this should refer to a folder such that this is the only wasm file in that folder.